tools/check-git-version.js

```diff
@@ -0,0 +1,25 @@
+const shell = require('shelljs');
+
+function checkGitVersion() {
+  try {
+    const gitVersion = shell
+      .exec('git --version', { silent: true })
+      .stdout.toString()
+      .slice(12)
+      .split('.');
+    if (
+      parseInt(gitVersion[0], 10) >= 2 &&
+      parseInt(gitVersion[1], 10) >= 22 &&
```

Comment:
What if it's version 3.0.0?

Reply:
Also, needs to be 2.33.0 of higher

Reply:
I added a GIT_MINIMUM_VESRION in the updated code and used it to compare to the user version we got..